components/rmrk/Gallery/History.vue

```diff
@@ -127,7 +127,7 @@ const components = {
 
 type TableRow = {
   Type: string
-  Item?: string // only in collection
+  Item?: object // only in collection
```

Comment:
```suggestion
  Item?: TableRowItem // only in collection
```
`object` is not a proper type 👀 

Please create type like 
```ts
type TableRowItem = {
  id: string
  name: string
}
```

Reply:
thanks for the review, updated the Type for TableRowItem 🙏🏻